source/reference/minio-mc/mc-mirror.rst

```diff
@@ -191,44 +173,70 @@ Parameters
    
    Overwrites object(s) on the :mc-cmd:`~mc mirror TARGET`.
 
+   For example, consider an active ``mc mirror --overwrite`` synchronizing content from Source to Destination.
+
+   If an object on Source changes, ``my mirror --overwrite`` synchronizes and overwrites any matching file on Destination.
+
+   Without ``--overwrite``, if an object already exists on the Destination, the mirror process fails to synchronize that object.
+   MinIO logs an error and continues to synchronize other objects.
+
 .. mc-cmd:: --preserve, a
    
 
-   Preserve file system attributes and bucket policy rules of the
-   :mc-cmd:`~mc mirror SOURCE` on the
-   :mc-cmd:`~mc mirror TARGET`.
+   Preserve file system attributes and bucket policy rules of the :mc-cmd:`~mc mirror SOURCE` on the :mc-cmd:`~mc mirror TARGET`.
 
 .. mc-cmd:: --region
    
 
-   Specify the ``string`` region when creating new bucket(s) on the
-   target. 
+   Specify the ``string`` region when creating new bucket(s) on the target. 
 
    Defaults to ``"us-east-1"``.
 
 .. mc-cmd:: --remove
    
 
-   Removes extraneous object(s) on the target. 
+   Removes object(s) on the Target that do not exist on the Source. 
+
+   Use the `--remove` flag to have the same list of objects on both Source and Target.
+
+   For example, objects A, B, and C exist on Source.
+   Objects C, D, and E exist on Target.
+
+   When running ``mc mirror --remove``, objects A and B copy to Target and objects D and E are removed from Target.
+   Since an object C already exists on both, nothing moves from Source to Target. 
+
+   After the action, only objects A, B, and C exist on both the Source and the Target.
+
+   Note: MinIO does not verify that the contents of object C are the same on both Source and Target, only that an object called `C` exists on both.
+   To make object C on Target the same as object C on Source, use  :mc-cmd:`~mc mirror --overwrite` or :mc-cmd:`~mc mirror --watch`.
```

Comment:
```suggestion
   To ensure objects on the Source and Destination match both names *and* content, use  :mc-cmd:`~mc mirror --overwrite` or :mc-cmd:`~mc mirror --watch`.
```

Reply:
Using `Target` instead of `Destination` to match the rest of this section.
Otherwise, 👍